docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,271 @@
+  - Customers using the VHD scripts must ensure they are following [Windows licensing requirements](https://www.microsoft.com/en-us/licensing/product-licensing/windows-server)
+- Wins is a third party dependency
+  - The project is under the Apache 2.0 License
+
+## Alternatives
+
+1. An alternative to using wins and daemonsets to do the configuration to download and configure components as services 
+   using the kubeadm pre/post commands. This would require the infrastructure providers to have the ability to pass 
+   configuration through the use of these commands which is already done today. During the Alpha phase with the pre/post 
+   scripts being developed by individual infra providers this will not be an issue.  With the move to Windows privileged
+   containers in Beta, this becomes a non issue as wins will no longer be required.
+
+1. Create a separate bootstrap provider for Windows.  This would require re-implementing a lot of the logic that is 
+   already in CABPK.  
+   When bugs are fixed or changes in behavior occur, Windows would risk being out of sync with the Linux implementation.
+
+1. Modified CABPK provider to have a different output format than cloud-init for Windows nodes.   With Cloudbase-init 
+   there are no requirements to change CABPK which makes the adaption for Windows straightforward. If we were to adapt 
+   the output format of CABPK there is a potential for introduction of bugs and variation in the logic that would be 
+   created for Windows nodes.  This would cause the Windows implementation to differ from others which could lead to 
+   confusion when debugging differences.
+
+## Upgrade Strategy
+
+Nodes that use this pattern will require the infrastructure to be immutable as specified by CAPI documentation.
+
+## Additional Details
+
+### Test Plan [optional]
+
+**Note:** *Section not required until targeted at a release.*
+
+Given there are no changes to the CAPI codebase, The testing plan would be left up to each infrastructure provider. 
+It would be recommended to leverage the existing upstream Windows tests to show that Windows nodes are operating 
+effectively.  If changes are required to the CAPI codebase unit tests and if appropriate e2e tests should be added.
+
+### Graduation Criteria [optional]
+
+**Note:** *Section not required until targeted at a release.*
+
+#### Alpha
+`PreKubeadmCommands`/`PostKubeadmCommands` command scripts are created per infrastructure provider
+Initially implemented with wins and pre/post commands
+Windows Packer script to create VHD with cloudbase-init added to the image builder scripts
+
+#### Beta
+Pre/Post commands moved to bootstrap provider
+Adopt privileged containers (dependant on Privileged containers KEP)
+kubeadm bootstrap token can be kept secret via multi-part mime documents for cloudbase-init. 
```

Comment:
If this is meant to be a list prepend `-`. It currently shows as a run off sentence.